ci-operator/step-registry/baremetal/lab/pre/firewall/baremetal-lab-pre-firewall-commands.sh

```diff
@@ -13,6 +13,25 @@ SSHOPTS=(-o 'ConnectTimeout=5'
   -o LogLevel=ERROR
   -i "${CLUSTER_PROFILE_DIR}/ssh-key")
 
+setup_proxy() {
+  proxy="$(<"${CLUSTER_PROFILE_DIR}/proxy")"
+  cat <<EOF > "${SHARED_DIR}/proxy-conf.sh"
+  export HTTP_PROXY=${proxy}
+  export HTTPS_PROXY=${proxy}
+  export NO_PROXY="localhost,127.0.0.1"
+
+  export http_proxy=${proxy}
+  export https_proxy=${proxy}
+  export no_proxy="localhost,127.0.0.1"
+EOF
+}
+
+if [ "${CLUSTER_WIDE_PROXY}" == "true" ]; then
+  # ipi-conf-proxy will run only if a specific file is found, see step code
+  cp "${CLUSTER_PROFILE_DIR}/proxy_private_url" "${SHARED_DIR}/proxy_private_url"
+  setup_proxy
```

Comment:
```suggestion
```

We need to run `setup_proxy` in any case (CLUSTER_WIDE_PROXY or DISCONNECTED), right?